Hyperledger_Fabric_Technical_Charter.md

```diff
@@ -1,116 +1,159 @@
 
-   5. The Project will operate in a transparent, open, collaborative, and ethical manner at all times. The output of all Project discussions, proposals, timelines, decisions, and status should be made open and easily visible to all. Any potential violations of this requirement should be reported immediately to the Series Manager.
+    3. **Ecosystem-Wide Voting Process**
 
-5. **Community Assets**
+        1.  **Initiation**: Proposals for ecosystem-wide decisions are reviewed by the TSC. If a formal vote is deemed necessary, the TSC initiates it, putting the matter to the TSC Members affiliated with all recognized and active Sub-Projects for their respective Sub-Project's collective vote.
 
-   1. LF Projects will hold title to all trade or service marks used by the Project (“Project Trademarks”), whether based on common law or registered rights.  Project Trademarks will be transferred and assigned to LF Projects to hold on behalf of the Project. Any use of any Project Trademarks by Collaborators in the Project will be in accordance with the license from LF Projects and inure to the benefit of LF Projects.  
+        2.  **Sub-Project Collective Vote Determination:**
+            
+            1. Each active Sub-Project contributes one collective vote (approve/disapprove) to an ecosystem-wide decision. This vote is determined by the TSC Members affiliated with that Sub-Project.
+            
+            2. **Internal Voting Eligibility:** Only the TSC Members affiliated with the Sub-Project are eligible to vote in the internal process that determines the Sub-Project's collective stance.
 
-   2. The Project will, as permitted and in accordance with such license from LF Projects, develop and own all Project GitHub and social media accounts, and domain name registrations created by the Project community.
+            3. **Internal Decision Mechanism:** Typically, a **Sub-Project's** collective vote is 'approve' if a majority of its participating affiliated TSC members vote to approve the proposal; otherwise, its collective vote is 'disapprove'. Alternatively, the **Sub-Project** may use its own documented procedures for its affiliated TSC members to determine its collective vote (approve/disapprove).
 
-   3. Under no circumstances will LF Projects be expected or required to undertake any action on behalf of the Project that is inconsistent with the tax-exempt status or purpose, as applicable, of the Joint Development Foundation or LF Projects, LLC.
+        3.  **Submission & Tally:** Each group of TSC Members affiliated with a Sub-Project submits their Sub-Project's collective vote to the overall TSC.
 
-6. **General Rules and Operations.** 
+        4. **Adoption & Announcement:** A proposal is adopted if approved by at least fifty percent of active Sub-Projects (based on their collective votes). The TSC tallies these collective votes and announces the outcome.
```

Comment:
```suggestion
4. **Adoption & Announcement:** A proposal is adopted if it receives:
   i. Approval from at least fifty percent of active Sub-Projects, AND
   ii. No explicit objections from any participating Sub-Project
   
Abstentions are permitted and do not count as objections. The TSC tallies these collective votes and announces the outcome.```

The current simple majority threshold (50%+) is insufficient for ecosystem governance and vulnerable to hostile control. Consider this scenario:

Current System: With 10 Sub-Projects, a proposal passes with 6 "Yes" votes, even if 4 vote "No"
Problem: A coordinated group controlling 6 Sub-Projects can force through any decision despite significant opposition

Reply:
@C0rWin the charter can always be amended. I am sure that new sub projects are not going to be added very often. Especially, when we have 10 sub-projects, we may need to conduct elections 

As @denyeart already pointed out, 50%+ is what followed in LFDT.